generators/init-product/templates/_gitignore

```diff
@@ -12,6 +12,7 @@
 !templates/web/deploy
 !templates/web/deploy/web
 !templates/web/deploy/web/*
```

Comment:
Could we remove this lines since the new line `!templates/web/deploy/**/*` is a generalization and coveres this case?

Reply:
no because you have to add really all folders; https://stackoverflow.com/questions/5533050/gitignore-exclude-folder-but-include-specific-subfolder